dbml-homepage/docusaurus.config.ts

```diff
@@ -28,8 +29,11 @@ const config: Config = {
       {
         docs: {
           sidebarPath: './configs/sidebars.ts',
-          breadcrumbs: false,
+          breadcrumbs: true,
```

Comment:
Why do we need to update this prop?

Reply:
I just enabled this bar
![Screenshot 2024-08-19 at 16 00 55](https://github.com/user-attachments/assets/480ee39c-0b88-4b02-bdbd-ed167eaec2fc)